samples/BikeSharingApp/Gateway/appsettings.Development.json

```diff
@@ -11,7 +11,7 @@
     "Services": {
       "Users": "users",
       "Bikes": "bikes",
-      "Reservations": "reservations",
+      "Reservations": "reservation",
```

Comment:
Could you please rename the "Reservations" as well? Keeping things consistent will ensure we don't make the same error in the future